metpy/calc/tests/test_thermo.py

```diff
@@ -17,6 +17,17 @@
 from metpy.units import units
 
 
+def assertIsUnitNan(value, units):
```

Comment:
Yeah - I was unsure of the spelling, but wanted to get this off my drive. I'll clean it up a bit.